python/feathub/processors/spark/dataframe_builder/spark_dataframe_builder.py

```diff
@@ -0,0 +1,160 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from typing import Dict, Tuple, Union
+
+from pyspark.sql import DataFrame as NativeSparkDataFrame
```

Comment:
In files like `spark_table.py`, both pandas DataFrame and spark DataFrame are used. Thus it might be better to make "DataFrame` refer to pandas DataFrame by default in Feathub and rename spark DataFrame.

Reply:
Sounds good.